frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,23 @@
+import { type IconNames } from '#libs/enums/enums.js';
+import { iconNameToPlainSvgMap } from '#libs/maps/icon-name-to-plain-svg.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  name: IconNames;
+  className?: string;
+};
+
+const Icon: React.FC<Properties> = ({ name, className = '' }) => {
+  const SelectedIcon = iconNameToPlainSvgMap[name];
+
+  const cssClass = className && styles[className] ? styles[className] : '';
+
```

Comment:
I think it's better to use getValidClassNames for this purpose. It could be something like this.
```suggestion
  const classNames = getValidClassNames(className, styles['default-className']);

```